### PR TITLE
Sighte adds Dublin

### LINKED ISSUE #29: sighte follows GitHub Flow
# Dublin, Ireland

- [ ] Create a **branch**
- [ ] Add some **commits**
- [ ] Open a *pull request*
- [ ] Collaborate
- [ ] `Merge` with 👍

### PR DESCRIPTION
**Pull request by sighte**
This pull request is to add Dublin, Ireland to the list of hometowns.

This PR closes #29.

### CODE REVIEW

File: sighte-dublin.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Dublin, Ireland
+
+### Restaurants
+
+- Borza
```
Comment (vijaybabu27) [inline sighte-dublin.md]: @sighte how about adding few more restaurants

Comment (sighte) [inline sighte-dublin.md]: I'll test a few out first 😄

Comment (sighte) [inline sighte-dublin.md]: > @sighte how about adding few more restaurants

@vijaybabu27 I thought of one more!

Comment (sighte) [inline sighte-dublin.md]: How about an approval? 👍

**Pull request merged**